responsemodifiers/headers.go

```diff
@@ -4,46 +4,43 @@ import (
 	"net/http"
 
 	"github.com/containous/traefik/config"
+	"github.com/containous/traefik/middlewares/headers"
 	"github.com/unrolled/secure"
 )
 
-func buildHeaders(headers *config.Headers) func(*http.Response) error {
+func buildHeaders(h *config.Headers) func(*http.Response) error {
```

Comment:
could you restore the variables name `headers` because it can be confused with a receiver name.

Reply:
No, because if we do, there is a naming conflict between `config.Headers` and `middlewares.headers`, resulting in the error:

`headers.NewHeader undefined (type *config.Headers has no field or method NewHeader)`

Reply:
Could we name it something else so there is no conflict? like `hdr`?

Reply:
ok